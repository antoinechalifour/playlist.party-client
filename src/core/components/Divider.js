import styled from 'styled-components'

const Divider = styled.hr`
  opacity: .25;
  margin: 16px 0;
`

export default Divider
