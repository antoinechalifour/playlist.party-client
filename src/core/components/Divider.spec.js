import React from 'react'
import renderer from 'react-test-renderer'
import Divider from './Divider'
import 'jest-styled-components'

describe('Divider', () => {
  it('Should render a divider', () => {
    const tree = renderer.create(<Divider />).toJSON()

    expect(tree).toMatchSnapshot()
  })
})
