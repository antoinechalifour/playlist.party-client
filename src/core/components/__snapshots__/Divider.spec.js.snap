// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Divider Should render a divider 1`] = `
.c0 {
  opacity: .25;
  margin: 16px 0;
}

<hr
  className="c0"
/>
`;
